next-tavla/app/(admin)/boards/components/TableRows/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { useSortBoardFunction } from '../../hooks/useSortBoardFunction'
+import { DEFAULT_BOARD_NAME } from 'Admin/utils/constants'
+import { Column } from '../Column'
+import { Fragment } from 'react'
+import { TBoard } from 'types/settings'
+import { DEFAULT_BOARD_COLUMNS, TBoardsColumn } from 'Admin/types/boards'
+import { intersection, uniq } from 'lodash'
+import { TTag } from 'types/meta'
+import { useSearchParam } from '../../hooks/useSearchParam'
+
+function TableRows({ boards }: { boards: TBoard[] }) {
+    const search = useSearchParam('search') ?? ''
+    const filter = useSearchParam('tags')?.split(',') ?? []
+    const columns =
+        useSearchParam('columns')?.split(',') ?? DEFAULT_BOARD_COLUMNS
+    const sortFunction = useSortBoardFunction()
+    const searchFilter = new RegExp(search, 'i')
+    return (
+        <>
+            {boards
+                .filter((board: TBoard) =>
+                    searchFilter.test(board?.meta?.title ?? DEFAULT_BOARD_NAME),
+                )
+                .filter(
+                    (board: TBoard) =>
+                        filter.length === 0 ||
+                        intersection(board?.meta?.tags ?? [], filter).length >
+                            0,
+                )
```

Comment:
Dette hadde kanskje vært enklere å forstå om man hadde definert de to filterne som funksjoner litt lenger opp